sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -1540,4 +1596,10 @@ public partial class SnapshotPolicyWeeklySchedule
         public int? SnapshotsToKeep { get { throw null; } set { } }
         public long? UsedBytes { get { throw null; } set { } }
     }
+    public partial class VolumeRelocationProperties
+    {
+        public VolumeRelocationProperties() { }
+        public bool? ReadyToBeFinalized { get { throw null; } }
```

Comment:
Could we rename this property to `IsReadyToBeFinalized`?

Reply:
We should probably be able to do that in autorest.md config right?